cloud/data-tiering/about-data-tiering.md

```diff
@@ -0,0 +1,106 @@
+
+*   **Data warehousing.** Access all your data without leaving Timescale Cloud.
+    Rather than running a separate system to tier and archive historical data,
+    move it to native object storage.
+
+*   **Transparent SQL queries.** You can interact with your data normally even
+    when it's distributed across different storage layers. Your hypertable is
+    spread across the layers, so queries and `JOIN`s work and fetch the same
+    data as usual.
+
+## Architecture
+
+Data tiering works by periodically and asynchronously moving older chunks to S3
+storage. There, it's stored in the Apache Parquet format, which is a compressed
+columnar format well-suited for S3. Data remains accessible both during and
+after migration.
+
+When you run regular SQL queries, a behind-the-scenes process transparently
+pulls data from wherever it's located: disk storage, object storage, or both.
+Various SQL optimizations limit what needs to be read from S3:
+
+*   Chunk exclusion avoids processing chunks that fall outside the query's time
+    window
+*   The database uses metadata about row groups and columnar offsets, so only
+    part of an object needs to be read from S3
+
+The result is transparent queries across standard PostgreSQL storage and S3
+storage, so your queries fetch the same data as before, with minimal added
+latency.
+
+For more about how data tiering works, see the
+[blog post on data tiering][blog-data-tiering].
+
+For more about using data tiering in your applications, see the
+[data tiering API documentation][data-tiering-api].
+
+## Limitations
+
+*   **Limited schema modifications.** Some schema modifications are not allowed
+    on hypertables with tiered chunks.
+
+    _Allowed_ modifications include: renaming the hypertable, adding columns
+    with `NULL` defaults, adding indexes, changing or renaming the hypertable
+    schema, and adding `CHECK` constraints. For `CHECK` constraints, only
+    untiered data is verified.
+
+    _Disallowed_ modifications include: adding a column with non-`NULL`
+    defaults, renaming a column, deleting a column, changing the data type of a
+    column, and adding a `NOT NULL` constraint to the column.
+
+*   **Limited data changes.** You cannot insert data into, update, or delete a
+    tiered chunk. These limitations take effect as soon as the chunk is
+    scheduled for tiering.
+
+*   **Inefficient query planner filtering for non-native data types.** The query
+    planner speeds up reads from object storage by using metadata to filter out
+    columns and row groups that don't satisfy the query. This works for all
+    native data types, but not for non-native types, such as `JSON`, `JSONB`,
+    and `GIS`.
+
+*   **Latency tradeoffs.** S3 has a much higher latency than EBS. Latency for
+    S3 is in the 100s of milliseconds vs 1-10ms latency for EBS volumes. That can
+    affect the execution time of queries, especially lighter queries in latency
+    sensitive environments.
```

Comment:
```suggestion
*   **Latency.** S3 has much higher latency than EBS. Latency for S3 is in 
     the 100s of milliseconds, compared to 1-10ms latency for EBS volumes. 
     This can affect the execution time of queries, especially lighter queries 
     in latency-sensitive environments.
```